website/docs/client/python.md

```diff
@@ -441,6 +441,94 @@ client = OpenLineageClient(transport=FileTransport(file_config))
 
 </Tabs>
 
+### Composite
+
+The `CompositeTransport` is designed to aggregate multiple transports, allowing event emission to several destinations sequentially. This is useful when events need to be sent to multiple targets, such as a logging system and an API endpoint, one after another in a defined order.
```

Comment:
Proposal:
The `CompositeTransport` is designed to combine multiple transports, allowing event emission to several destinations. This is useful when events need to be sent to multiple targets, such as a logging system and an API endpoint. The events are delivered sequentialy - one after another in a defined order.